### DISCUSSION TITLE
How to invert release title regex?

### DISCUSSION THREAD

**Discussion by stepanzak** [ImranR98/Obtainium#1635] [Q&A] [answered]
I use [Lawnchair](https://github.com/LawnchairLauncher/lawnchair) beta, but I don't want to install the nightly version. (It has different signature than the beta ones.)
How can I filter releases so it ignores any that has "Nightly" in the title.

**Comment by ImranR98** [comment, marked as answer]
There's no invert option for that filter, but you could use `[0-9]` to filter out titles that don't have any numbers (looks like nightly is the only one that  doesn't).

**Comment by callmeTayyy** [comment]
@stepanzak you can achieve this by:
* turning on 'include prereleases'
* setting 'Filter release titles...' as `Beta` or `[Bb]eta`
